bench_dataset.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import json, os, argparse
```

Comment:
Avoid unused package importing;
Would it be better to add a main function mark "if __name__ == "__main__""?